client/.storybook/config.js

```diff
@@ -1,9 +1,9 @@
-import { configure } from '@storybook/react';
+import { addDecorator, configure } from '@storybook/react';
+import React from 'react';
 
-// const loadStories = () => {
-//   const req = require.context('../src', true, /\.story\.(js|mdx)$/);
-//   req.keys().forEach(filename => req(filename));
-// };
+addDecorator(Story => <Story />);
```

Comment:
This was the winner right here.

Reply:
I knew I was on the right track with that but I couldn't get all the way there.